sdb/locator.py

```diff
@@ -44,7 +44,9 @@ def caller(self, objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
         based on the type of the input we receive.
         """
 
-        out_type = sdb.prog.type(self.output_type)
+        out_type = None
+        if self.output_type is not None:
+            out_type = sdb.prog.type(self.output_type)
```

Comment:
This would probably need to be updated to use the appropriate API call from `target.py`